src/frontend/shortlist/src/views/ShareView.vue

```diff
@@ -40,76 +43,69 @@ export default {
   data() {
     let dataSuccess = false;
     let shortlistData = {};
+    let bgcolor = "#030303";
     return {
       dataSuccess,
       shortlistData,
+      bgcolor,
     };
   },
 };
 </script>
 
 <template>
   <div v-if="dataSuccess">
-    <h1>Report Card</h1>
-    <div class="school-simple-container">
+    <label class="share-shortlist-title">
+      <MaterialIcon
+        :src="this.shortlistData[0].settings.icon.value"
+        size="20"
+        style="color: white; size: 30px"
+      />
+      {{ this.shortlistData[0].settings.name }} Shortlist Report
+    </label>
+    <div class="report-school-cards">
       <template
-        v-for="(schoolData, schoolIndex) in shortlistData[0].schools"
+        v-for="schoolData in shortlistData[0].schools"
         :key="schoolData"
       >
-        <div class="school-simple-name-row">
-          <div class="school-simple-name-name">
-            {{ schoolData.schoolMetadata.name }}
-          </div>
-          <div class="school-simple-name-borough">
-            {{ getBorough(schoolIndex) }}
-            <br />Phone: {{ schoolData.schoolMetadata.phone }} <br />Email:
-            {{ schoolData.schoolMetadata.email }} <br />School Website:
-            <a :href="schoolData.schoolMetadata.url">{{
-              schoolData.schoolMetadata.url
-            }}</a>
-          </div>
-        </div>
-        <div class="school-simple-dim-container">
-          {{ schoolData.schoolMetadata.desc }}
-        </div>
```

Comment:
Seems most card design are been removed instead just kept cards here, it's not what we talked about in design. It suppose to have this report, but with a button on each school name to open up its simple school card to view school quality

Reply:
I hope you have considered that the cards on the share page are just like with your decided details. I am unsure about buttons may be you can discuss that in the team channel.

Reply:
This theme is allowing users to see each and every 4 schools in one sight. Also, it includes settings of the user from categorize. I created the printable idea and removed the dimensions tap view; however, after discussing with @adelaidemz  we agreed on adding that back so that users can review that too.

Reply:
How about this, keep the current format that I put in there, just add simple card with qualities, is that clear?

Reply:
It's an added feature Adelaide thinks can be added to show school qualities too in a card format if they want to take a look, NOT a design overhaul on the current report format

Reply:
Please get this idea right? If you are still unclear, please schedule a meeting.

Reply:
Let's discuss and get clear on meeting @yalexublue @adelaidemz.